src/transpiler/transpiler.ts

```diff
@@ -133,9 +133,9 @@ function wrapWithPreviouslyDeclaredGlobals(
   nativeStorage: NativeStorage,
   globalIds: NativeIds
 ) {
-  let currentVariableScope = nativeStorage.globals!.previousScope
-  let timesToGoUp = 1
-  let wrapped = create.blockStatement(statements)
+  let currentVariableScope = nativeStorage.globals!
+  let timesToGoUp = 0
+  let wrapped = statements
```

Comment:
@openorclose this seems to have removed a block layer. Looking at the first few edited snapshots, it looks like it was useless to begin with? Do you know why we need this layer?